content/blog/understanding-the-dom/index.md

```diff
@@ -0,0 +1,520 @@
+
+
+
+# Using The Correct Tags {#accessibility}
+
+HTML, as a specification, has tons of tags that can be used at one's disposal. These tags contain various metadata internally to the browser to provide information about what you want to render in the DOM. This metadata can then be handled by the browser how it sees fit; it may apply default CSS styling, it may change the default interaction the user has with it, or even what behavior that element has upon clicking on it (in the case of a button in a form). 
+
+Some of these tag defaults are part of the specification, while others are left up to the browser vendor to decide. This is why, in many instances, developers may choose to use something like [`normalize.css`](https://github.com/necolas/normalize.css/) to set all of the element CSS defaults to an explicit set of defaults. Doing so can avoid having the UI of a webpage look different from browser-to-browser thanks to deviations on default CSS styling on specific tags.
+
+This metadata is also why it's so important that your application utilizes the expected HTML tags and not simply default to `<div>`s with CSS or JavaScript applied to simulate other items. Two of the biggest advantages to responsibly utilizing the metadata system the browser has built into it by using the correct tags are search engine optimization (SEO) and accessibility.
+
+Take the following example:
+
+```html
+<div>
+	<div>Bananas</div>
+	<div>Apples</div>
+	<div>Oranges</div>
+</div>
+```
+
+In this example, your browser only knows that you're looking to display text on-screen. If someone utilizing a screen reader reaches the site, the browser doesn't know that it should inform them that there are three items in a list (something that people with low vision would greatly value to know, in order to tab through the list effectively) as you've done nothing to inform the user that it is a list of items: only that it's a set of `<div>` generic containers. 
+
+Likewise, when Google's robots walk through your site, they won't be able to parse that you're displaying lists to your users. As a result, your search rating for "list of best places" might be impacted since the site doesn't appear to contain any list at all.
+
+What can be done to remediate this? Well, by utilizing the proper tags, of course!
+
+```html
+<ol>
+	<li>Bananas</li>
+	<li>Apples</li>
+	<li>Oranges</li>
+</ol>
+```
+
+In this example, both the browsers as well as Google's scraper bots are able to discern that this is a list with three list items within it.
+
+> While there ARE tags that may potentially  impact SEO somewhat significantly, it's unlikely `ul` and `li` would significantly impact your SEO scores.
+>
+> Needless to say, it's still good to use symantic (correctly tagged) HTML as there are users that use screen-readers that will benefit greatly from these minor changes. Additionally, it can make code more readable and parsable with automated tools
```

Comment:
```suggestion
> Needless to say, it's still good to use semantic (correctly tagged) HTML as people that use screen-readers and other assistive technologies benefit greatly from these minor changes. Additionally, it can make code more readable and parsable with automated tools.
```